geometry_msgs/Pose pose
string frame
---
bool result
